Mathlib/GroupTheory/Perm/Finite.lean

```diff
@@ -248,6 +248,51 @@ theorem support_pow_coprime {σ : Perm α} {n : ℕ} (h : Nat.Coprime n (orderOf
     le_antisymm (support_pow_le σ n)
       (le_trans (ge_of_eq (congr_arg support hm)) (support_pow_le (σ ^ n) m))
 
+lemma ofSubtype_support_disjoint {σ : Perm α} (x : Perm (Function.fixedPoints σ)) :
+    _root_.Disjoint x.ofSubtype.support σ.support := by
+  rw [Finset.disjoint_iff_ne]
+  rintro a ha b hb rfl
+  rw [mem_support] at ha hb
+  exact ha (ofSubtype_apply_of_not_mem x (mt Function.mem_fixedPoints_iff.mp hb))
+
+open Subgroup
+
+lemma disjoint_of_disjoint_support {H K : Subgroup (Perm α)}
+    (h : ∀ a ∈ H, ∀ b ∈ K, _root_.Disjoint a.support b.support) :
```

Comment:
Style question for here and below: is this "disjoint on support" better phrased this fully applied way, or as talking about the image of the set/set-like under `support` in a point-free style?

Reply:
I am sorry, but I don't understand the question !

Reply:
Never mind, we don't have a "take two sets and describe disjoint aspect between pairs taken one from each"